packages/api/src/schema/schema.yaml

```diff
@@ -469,6 +469,27 @@ components:
           example: 66E2161C-7670-4D05-B71D-DA2D6979556F
           index: true
 
+    region:
+      type: object
+      table: regions
+      required:
+        - region
+        - orchestrators
+      properties:
+        region:
+          type: string
+          description: region name
+          example: ber
+          unique: true
+          index: true
+        orchestrators:
+          type: array
+          items:
+            type: object
```

Comment:
Should add `additionalProperties: false` here or you can add whatever else you want. Fun! Also should do:
```
required:
- address
```
So that we don't accept empty objects.